netbox/project-static/styles/sidenav.scss

```diff
@@ -78,6 +78,7 @@
   background-color: var(--nbx-sidebar-bg);
   border-right: 1px solid $border-color;
   transition: $transition-100ms-ease-in-out;
+  transition-delay: 200ms;
```

Comment:
After some brief experimentation, this feels a bit sluggish. 100ms feels more responsive IMO while still introducing sufficient delay in trigger the transition.

Reply:
100ms it is